tests/MongoDB.Driver.Tests/JsonDrivenTests/JsonDrivenWithThreadTest.cs

```diff
@@ -0,0 +1,59 @@
+﻿/* Copyright 2020-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using MongoDB.Bson;
+using MongoDB.Driver.Core.Misc;
+
+namespace MongoDB.Driver.Tests.JsonDrivenTests
+{
+    public class JsonDrivenWithThreadState
+    {
+        #region static
+        public static string Key => nameof(Tasks);
+        #endregion
+
+        public ConcurrentDictionary<string, Task> Tasks { get; } = new ConcurrentDictionary<string, Task>();
+    }
+
+    public abstract class JsonDrivenWithThreadTest : JsonDrivenTestRunnerTest
+    {
+        protected string _name;
+        protected readonly JsonDrivenWithThreadState _testState;
+
+        public JsonDrivenWithThreadTest(
+            JsonDrivenTestsStateHolder stateHolder,
+            IJsonDrivenTestRunner testRunner,
+            Dictionary<string, object> objectMap) : base(testRunner, objectMap)
```

Comment:
Move to new line:

    : base(testRunner, objectMap)

Reply:
done